packages/core/admin/admin/src/hooks/marketplace/useFetchMarketplaceProviders/index.js

```diff
@@ -1,6 +1,26 @@
 import { useQuery } from 'react-query';
 import { useNotification } from '@strapi/helper-plugin';
-import { fetchMarketplaceProviders } from './utils/api';
+import qs from 'qs';
+import { MARKETPLACE_API_URL } from '../constants';
+
+const fetchMarketplaceProviders = async (params = {}) => {
+  try {
+    const queryString = qs.stringify(qs.parse(params));
+    const res = await fetch(`${MARKETPLACE_API_URL}/providers?${queryString}`);
+
+    if (!res.ok) {
+      throw new Error('Failed to fetch marketplace providers.');
+    }
+
+    const data = await res.json();
+
+    return data;
```

Comment:
I think you could just `return res.json()` if you wanted 🤷‍♂️

Reply:
ok 👍🏽

Reply:
Hey @simotae14 thanks for the update but you might want to double check Gustav's point above :)

Reply:
ok, done